tests/api/test_deployments.py

```diff
@@ -1,3 +1,14 @@
+from connect.eaas.core.constants import PROXIED_CONNECT_API_ATTR_NAME
+
+
+def test_proxied_connect_endpoints(api_client):
+    proxied = getattr(api_client._webapp_class, PROXIED_CONNECT_API_ATTR_NAME)
+    assert isinstance(proxied, dict)
+    assert proxied == {
+        '/public/v1/media': 'edit',
+    }
+
```

Comment:
I also propose to add one more test to check the resulting url, because theoretically proxied_connect_api may change in future.
It's not crucial to implement it now, but I think it's good to add it one day. Do you agree?

Reply:
The test here is to keep track of proxied endpoints that webapp currently have. Im not sure if full url could be tested from the api_client, because `proxied_connect_api` is a configuration for the runner.